dev/stage-binaries.sh

```diff
@@ -18,9 +18,14 @@
 # under the License.
 #
 
+SCALA_VERSION=2.12
 FLINK_VERSIONS=1.13,1.14,1.15
 SPARK_VERSIONS=2.4,3.0,3.1,3.2
 HIVE_VERSIONS=2,3
 
-./gradlew -Prelease -DflinkVersions=$FLINK_VERSIONS -DsparkVersions=$SPARK_VERSIONS -DhiveVersions=$HIVE_VERSIONS publishApachePublicationToMavenRepository
+./gradlew -Prelease -DscalaVersion=$SCALA_VERSION -DflinkVersions=$FLINK_VERSIONS -DsparkVersions=$SPARK_VERSIONS -DhiveVersions=$HIVE_VERSIONS publishApachePublicationToMavenRepository
+
+# Also publish Scala 2.13 Artifacts for versions that support it.
+# Flink does not yet support 2.13 (and is largely dropping a user-facing dependency on Scala). Hive doesn't need a Scala specification.
+./gradlew -P release -DscalaVersion=2.13 :iceberg-spark:iceberg-spark-3.2_2.13:publishApachePublicationToMavenRepository :iceberg-spark:iceberg-spark-extensions-3.2_2.13:publishApachePublicationToMavenRepository :iceberg-spark:iceberg-spark-runtime-3.2_2.13:publishApachePublicationToMavenRepository
```

Comment:
In the future we'll need to split this out into a matrix of Scala / Spark versions, but for now I think simpler is better as Scala 2.13 is really only a Spark 3.2 concern.

Reply:
There's a space in this line `-P release` that isn't in the one above. Is that correct? If we don't get the `release` flag set correctly then the artifact will go to the wrong Nexus repository.

Reply:
I went ahead and changed this just in case, since I remembered I can edit for simple things.

Reply:
Good catch @rdblue!